Progress not toggling on some habits

My User ID: fc1b3c1a-fb3c-4070-acf3-7acf64432156
And I'm currently in the most recent version of Chrome, operating in Windows 7.

About half of my habits are currently not doing anything when I click on the progress bar. The other half, and my dailies, seem to work fine, but these just don't react. I don't get any error messages when this happens.

Hi there!

Here's the full error report:

Given URL is not allowed by the Application configuration.: One or more of
the given URLs is not allowed by the App's settings.  It must match the
Website URL or Canvas URL, or the domain must be a subdomain of one of the
App's domains.
app-ef854652.js:6 Error: [$sce:unsafe] Attempting to use an unsafe value in
a safe context.
http://errors.angularjs.org/1.3.9/$sce/unsafe
    at https://habitrpg.com/app-ef854652.js:4:21718
    at n (https://habitrpg.com/app-ef854652.js:7:18373)
    at m (https://habitrpg.com/app-ef854652.js:7:19869)
    at Object.d.(anonymous function) [as getTrustedHtml](https://habitrpg.com/app-ef854652.js:7:20881)
    at Object.f (https://habitrpg.com/app-ef854652.js:8:8520)
    at Object.h [as fn](https://habitrpg.com/app-ef854652.js:7:6319)
    at j.$digest (https://habitrpg.com/app-ef854652.js:7:13775)
    at j.$apply (https://habitrpg.com/app-ef854652.js:7:15329)
    at s (https://habitrpg.com/app-ef854652.js:6:11271)
    at w (https://habitrpg.com/app-ef854652.js:6:12887)
https://googleads.g.doubleclick.net/pagead/ads?client=ca-pub-32423502438277
…l=NS&ppjl=t&fu=16&bc=1&ifi=6&xpc=4YclEutD77&p=https%3A//habitrpg.com&dtd=M
Failed to load resource: the server responded with a status of 400 (Bad
Request)
app-ef854652.js:6 Error: [$sce:unsafe] Attempting to use an unsafe value in
a safe context.
http://errors.angularjs.org/1.3.9/$sce/unsafe
    at https://habitrpg.com/app-ef854652.js:4:21718
    at n (https://habitrpg.com/app-ef854652.js:7:18373)
    at m (https://habitrpg.com/app-ef854652.js:7:19869)
    at Object.d.(anonymous function) [as getTrustedHtml](https://habitrpg.com/app-ef854652.js:7:20881)
    at Object.f (https://habitrpg.com/app-ef854652.js:8:8520)
    at Object.h [as fn](https://habitrpg.com/app-ef854652.js:7:6319)
    at j.$digest (https://habitrpg.com/app-ef854652.js:7:13775)
    at j.$apply (https://habitrpg.com/app-ef854652.js:7:15329)
    at HTMLAnchorElement.<anonymous> (
https://habitrpg.com/app-ef854652.js:8:10466)
    at HTMLAnchorElement.n.event.dispatch (
https://habitrpg.com/app-ef854652.js:2:6444)
https://googleads.g.doubleclick.net/pagead/ads?client=ca-pub-32423502438277
…l=NS&ppjl=t&fu=16&bc=1&ifi=7&xpc=sZjoTiMkYW&p=https%3A//habitrpg.com&dtd=M
Failed to load resource: the server responded with a status of 400 (Bad
Request)
https://googleads.g.doubleclick.net/pagead/ads?client=ca-pub-32423502438277
…l=NS&ppjl=t&fu=16&bc=1&ifi=8&xpc=yZhkp9Khvy&p=https%3A//habitrpg.com&dtd=M
Failed to load resource: the server responded with a status of 400 (Bad
Request)
https://apis.google.com/u/0/se/0/_/+1/fastbutton?usegapi=1&size=medium&hl=e…_1421692193650&parent=https%3A%2F%2Fhabitrpg.com&pfname=&rpctoken=19390430
Failed to load resource: net::ERR_CACHE_MISS
https://www.facebook.com/connect/ping?client_id=172525162793917&domain=habi…4%26relation%3Dparent&response_type=token%2Csigned_request%2Ccode&sdk=joey
Failed to load resource: net::ERR_CACHE_MISS
https://www.facebook.com/plugins/like.php?action=like&app_id=17252516279391…=en_US&ref=.VL1NIP2r2gk.like&sdk=joey&send=false&show_faces=false&width=90
Failed to load resource: net::ERR_CACHE_MISS
https://googleads.g.doubleclick.net/pagead/ads?client=ca-pub-32423502438277
…l=NS&ppjl=t&fu=16&bc=1&ifi=7&xpc=sZjoTiMkYW&p=https%3A//habitrpg.com&dtd=M
Failed to load resource: the server responded with a status of 400 (Bad
Request)
https://googleads.g.doubleclick.net/pagead/ads?client=ca-pub-32423502438277
…l=NS&ppjl=t&fu=16&bc=1&ifi=8&xpc=yZhkp9Khvy&p=https%3A//habitrpg.com&dtd=M
Failed to load resource: the server responded with a status of 400 (Bad
Request)

On Fri, Jan 16, 2015 at 12:42 PM, Matteo Pagliazzi <notifications@github.com

> wrote:
> 
> @Bewildebeest https://github.com/Bewildebeest thanks for the report.
> Could you please open HabitRPG in Google Chrome, then open the console
> Ctrl+Shift+J and click the progress the progress bar. If any error appears
> in the console, you should report it here.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/HabitRPG/habitrpg/issues/4521#issuecomment-70291374.

--- a/public/js/app.js
+++ b/public/js/app.js
@@ -1,7 +1,7 @@
 "use strict";
 
 window.habitrpg = angular.module('habitrpg',
-    ['ui.bootstrap', 'ui.keypress', 'ui.router', 'chieffancypants.loadingBar', 'At', 'infinite-scroll', 'ui.select2', 'angular.filter', 'ngResource'])
+    ['ui.bootstrap', 'ui.keypress', 'ui.router', 'chieffancypants.loadingBar', 'At', 'infinite-scroll', 'ui.select2', 'angular.filter', 'ngResource', 'ngSanitize'])
 
   // @see https://github.com/angular-ui/ui-router/issues/110 and https://github.com/HabitRPG/habitrpg/issues/1705
   // temporary hack until they have a better solution

